builds/misc/images.yaml

```diff
@@ -7,70 +7,27 @@ pr: none
 
 jobs:
 ################################################################################
-  - job: linuxARM64
+  - job: linux_ARM64
```

Comment:
`linux_dotnet_projects_arm64` maybe?

Reply:
Sounds good - I also rearranged the dependencies for the manifest build so that similarly named jobs are grouped.